src/Config.js

```diff
@@ -453,6 +455,12 @@ export class Config {
     }
   }
 
+  static validateDefaultLimit(defaultLimit) {
+    if (defaultLimit <= 0) {
```

Comment:
It needs to validate the type as well, you could use `isNaN(defaultLimit)`

Reply:
@mtrezza , if I'm not mistaken, you had already asked for additional validations here and I spent time implementing them, but then discovered that they aren't needed.
https://github.com/parse-community/parse-server/pull/8152/files#r966496139
I think this is a clear indication that this admittedly simple PR has been through way too many change requests.

Reply:
As I explained in the response for your previous request to add additional validations, I couldn't find a way to test this addition because, it seems, another component in the system already performs the checks you're looking for and throws the respective errors. The granularity of these tests (which I didn't decide upon) doesn't allow for testing this class in isolation, and, given the desired behvior is already implemented, I find it odd you're insisting on duplicating it.
In any case, you should've explained all this 18 days ago, when I responded to your change request, not circle back to it now.

> Remember that your "simple PR" will be pushed to thousands of deployments and developers; there are some quality aspects we need to take into account, beyond just adding the feature.

Thanks for lecturing me on the importance of quality. 

Allow me to circle back to what I wrote several days ago: I tried to do the "right thing" with this addition and contribute back to the project, but I feel that you're dragging this thing and nitpicking now. I tried to be a "good citizen", but I'm disappointed and demotivated by how this thing goes. So, here's an idea: you know the project much better and you'll be 10 times more effective than me in adding anything to this PR. So, if you find this validation so crucial, then, maybe, instead of doing this back-and-forth in comment times and again, just go ahead an add these several lines of code?

I feel that I spent way too much time on this PR. It would be much faster for us to just set limits manually everywhere, but I decided to try to do the right thing, so here we are. Now, I'm done. You're welcome to change this PR and merge it, or decline it altogether.

Reply:
> I couldn't find a way to test this addition because, it seems, another component in the system already performs the checks you're looking for

If Parse Server already habdled this and throws an error when setting anything else than a number then we don't need a validation. Is that the case?